glossary.asciidoc

```diff
@@ -19,14 +19,14 @@ Account::
     An object containing an address, balance and nonce, and optional storage and code. An account can be a contract account or an EOA (externally owned account).
 
 Address::
-    Most generally, this is the place where you receive transactions on the blockchain. More specifically, it is the right most 160-bits of a Keccak hash of an ECDSA public key.
+    Most generally, this is the place where you receive transactions on the blockchain or the source from where you send gas and payload to another address (contract or EOA account). More specifically, it is the right most 160-bits of a Keccak hash of an ECDSA public key.
 
 Assert::
     In Solidity assert(false) compiles to *0xfe*, which is an invalid opcode, using up all remaining gas, and reverting all changes.
     When an assert() statement fails, something very wrong and unexpected shoud be happening, and you will need to fix your code.
     You should use assert to avoid conditions which should never, ever be possible.
 
-// What is the case convention - bitcoin or Bitcoin?
+// What is the case convention - bitcoin or Bitcoin? Perhaps Bitcoin for the crypto/protocol/network and bitcoin for the actual coins.
```

Comment:
Generally, I have used lower case for bitcoin throughout my previous book. But since I make the distinction between Ethereum/ether, I should also use Bitcoin/bitcoin. This needs a whole new issue to fix everywhere